ultimatejobweb/views.py

```diff
@@ -33,4 +34,4 @@ def sign_in(request):
 
 
 def available_jobs(request):
-    return render(request, r"available_jobs.html")
+    return render(request, r"available_jobs.html", {'job': job})
```

Comment:
You're passing in the jobs model class. Generally you would want to use a filter or query. For example, if you want to show all the jobs in the system you would pass in jobs.objects.all()
(Though as I commented in #75, the class name should be Jobs so that you know its a class)